include/Pattern.h

```diff
@@ -181,6 +183,12 @@ class PatternView : public TrackContentObjectView
 	QColor const & getMutedNoteBorderColor() const { return m_mutedNoteBorderColor; }
 	void setMutedNoteBorderColor(QColor const & color) { m_mutedNoteBorderColor = color; }
 
+	QColor color() const
+	{
+		return( m_pat->color() );
+	}
+	void setColor( QColor _new_color );
```

Comment:
Please extract these functions into TrackContentObjectView, since they're identical for each subclass.

Reply:
Done.